### DISCUSSION TITLE
Security Onion 2.3.2 MSSP (multi-tenancy) support

### DISCUSSION THREAD

**Discussion by AnkitSlashKarn** [Security-Onion-Solutions/securityonion#2974] [Unsupported Versions] [answered]
Does Security Onion 2.0 (2.3.2) support the MSSP model for a 'Security Operation Center as a Service' kind of a model? If no, is there any plan to offer it in the coming updates to this version in the near future?

**Comment by AnkitSlashKarn** [comment]


**Reply by AnkitSlashKarn** [reply]
@defensivedepth  Would be glad if you could answer this one :)

**Comment by AnkitSlashKarn** [comment]
@defensivedepth Would be glad if you could answer this one :)

**Comment by TOoSmOotH** [comment, marked as answer]
There is currently no support for multi-tenancy. RBAC is something we talked about in the blog post below that might make this a possibility:

https://blog.securityonion.net/2020/12/security-onion-2-in-2020-and-2021.html
